### DISCUSSION TITLE
add sort order (ascending / descending) for music provider 'Spotify'

### DISCUSSION THREAD

**Discussion by darkman242** [music-assistant/support#3098] [Feature requests and ideas]
please add the ascending / descending sort order for the music provider Spotify so that the current albums can also be at the top. thanks

**Comment by OzGav** [comment]
Not sure what you are asking for here. The album view has many sort options.

**Comment by darkman242** [comment]
if I select the Album item in the main menu - yes. but if I select the Spotify account under the Browse tab and then the artist there - no.

Album Tab
<img width="1491" alt="Album Tab" src="https://github.com/user-attachments/assets/8a18b5cf-bf44-4730-8451-c974acb1f484">

BrowseTab -> Spotify -> Artist
<img width="1491" alt="Browse Tab" src="https://github.com/user-attachments/assets/3ea541e7-46ba-46eb-97b9-9fe4e010a409">

**Reply by OzGav** [reply]
OK the browse view is not really designed to be casually used. You should be using the main views.

**Reply by darkman242** [reply]
when I go to the artist tab, the sorting of the albums is still restricted

artist tab -> album
<img width="1491" alt="artist tab" src="https://github.com/user-attachments/assets/123b793d-51e4-4770-a9ef-0f2f33b2a0ff">

**Reply by OzGav** [reply]
You will need to be more specific about what you are requesting in order for others to understand and vote for it.

**Reply by OzGav** [reply]
The Artist-Album sort list now has more options. This is considered implemented
